src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirRequestContextDocumentClientExtensions.cs

```diff
@@ -40,6 +40,10 @@ public static void ProcessException(this IFhirRequestContext requestContext, Exc
                 {
                     throw new RequestRateExceededException(dce.RetryAfter);
                 }
+                else if (dce.Message.Contains("Invalid Continuation Token", StringComparison.OrdinalIgnoreCase))
```

Comment:
Only feedback I had was to ask if there was anything in `GetSubStatusCode` that could help with detecting this?

Reply:
https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.CosmosDb/Features/Storage/DocumentClientExceptionExtensions.cs

Reply:
Looking at the values in the ResponseHeaders through the debugger I don't see any value labeled as a sub-status, or any value that would help with detecting this.